models/src/main/scala/it/pagopa/interop/purposemanagement/model/purpose/PersistentPurpose.scala

```diff
@@ -15,5 +15,7 @@ final case class PersistentPurpose(
   description: String,
   riskAnalysisForm: Option[PersistentRiskAnalysisForm],
   createdAt: OffsetDateTime,
-  updatedAt: Option[OffsetDateTime]
+  updatedAt: Option[OffsetDateTime],
+  isFreeOfCharge: Boolean,
```

Comment:
@beetlecrunch this requires a sanification on the read model because this model is used to deserialize results